UPGRADE.md

```diff
@@ -34,6 +34,10 @@ After bundle install, you should also install npm packages:
 npm install
 ```
 
+This release changes how cookies are handled. Previously, all cookies were shared between different subdomains i.e. domain of cookies was sharetribe.com. Therefore, it was possible to log in to foo.sharetribe.com and log in automatically also to bar.sharetribe.com. Recently, the domain model was updated so that person can belong only to one marketplace. This caused an issue where logging in to foo.sharetribe.com and then visiting in bar.sharetribe.com logged user out of foo.sharetribe.com, because session was shared but the logged in user wasn't a member of bar.sharetribe.com.
```

Comment:
Even though the description is very nice I think it's unnecessary in upgrade guide. The paragraph below is to the point and describes the actions that need to be taken. In addition, multi marketplace setup is not part of open source target so no need to go into detail to explain it. Sorry if I was unclear in my earlier comment.